pandas/io/pickle.py

```diff
@@ -33,7 +35,34 @@ def to_pickle(obj, path, compression='infer', protocol=pkl.HIGHEST_PROTOCOL):
         .. [1] https://docs.python.org/3/library/pickle.html
         .. versionadded:: 0.21.0
 
-
+    Examples
+    --------
+    >>> original_df = pd.DataFrame({"foo": range(5), "bar": range(5, 10)})
+    >>> original_df
+       foo  bar
+    0    0    5
+    1    1    6
+    2    2    7
+    3    3    8
+    4    4    9
+    >>> pd.to_pickle(original_df, "./dummy.pkl")
+
+    >>> unpickled_df = pd.read_pickle("./dummy.pkl")
+    >>> unpickled_df
+       foo  bar
+    0    0    5
+    1    1    6
+    2    2    7
+    3    3    8
+    4    4    9
+
+    See Also
```

Comment:
`See Also` should go before the `Examples`.

Reply:
corrected.

Reply:
Did you push your latest changes?

Reply:
yes.

Reply:
Just because you mentioned you corrected it, but the see also is still after the examples?

Reply:
oh sorry, just missed this one and did others.